SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,75 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-02-16T01:00:00.000Z'
```

```suggestion
  expiration-date: '2025-02-14T01:00:00.000Z'
```

Should be max 1 year from last-reviewed